src/models/multiChart.js

```diff
@@ -19,7 +19,12 @@ nv.models.multiChart = function() {
         useVoronoi = true,
         interactiveLayer = nv.interactiveGuideline(),
         useInteractiveGuideline = false,
-        legendRightAxisHint = ' (right axis)'
+        legendRightAxisHint = ' (right axis)',
+        showY1Axis = true,
+        showY1AxisZero = false,
+        showY2Axis = true,
+        showY2AxisZero = false,
```

Comment:
It looks like multiChart is missing the forceX and forceY options that exist on most of the other charts, it should match this same style instead of inventing this separate (and more limited) option I think.